Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx

```diff
@@ -95,10 +87,14 @@ export interface ContentProps {
   setSelectedTriggers: (selectedTriggers: any[]) => void;
   setSkillManifest: (_: Partial<SkillManifestFile>) => void;
   schema: JSONSchema7;
+  selectedDialogs: any[];
+  selectedTriggers: any[];
   skillManifests: SkillManifestFile[];
   value: { [key: string]: any };
   onChange: (_: any) => void;
   projectId: string;
+  callers: string[];
+  setCallers: (callers: string[]) => void;
```

Comment:
```onUpdateCallers```, this is a callback not an imperative function to be called, convention for callbacks is ```onVerbSomething```

Reply:
Sure. fixed